sdk/src/driftClient.ts

```diff
@@ -154,8 +155,8 @@ export class DriftClient {
 		);
 
 		this.authority = config.authority ?? this.wallet.publicKey;
-		const subAccountIds = config.subAccountIds ?? [0];
-		this.activeSubAccountId = config.activeSubAccountId ?? subAccountIds[0];
+		this.activeSubAccountId = config.activeSubAccountId;
```

Comment:
why'd you get rid of the `?? [0]`?

Reply:
this way it will start undefined if not specified, then the call to `addUsers` will know no default is set and use the first user

Reply:
i think this might break some backwards compatibility since not every user will call `addUsers`

Reply:
bump on this